### ISSUE TITLE
Ammunition removal problems, unfireable

### ISSUE THREAD

**Issue by svalbardcaretaker** [crawl/crawl#2439]
Version: 0.29-a0-187-ga23d9638d5

Linux Console

I noticed this on wanderer starts. If one starts with a ranged weapon equipped, it is currently not possible to quiver the ranged weapon. TAB autoattack still works, but only as long as autofight allows that, after the autofight_stop treshold is reached its not possible to fire the ranged weapon!  Neither via [q] or [f] nor [TAB]. 

Regards

**Comment by liammcd** [comment]
You can fire with [v]

**Comment by rawlins** [comment]
Yes, this is not a bug in the sense that newlaunchers intentionally don't currently use the quiver system at all, and fire via `v` like a polearm (also `F` in current trunk). We may still change the behavior.
